### ISSUE TITLE
External Card/project link and goto button

### ISSUE THREAD

**Issue by krestyanikov** [tobybatch/TrelloBundle#11]
Hello @tobybatch 

Currently, ChromeKimaiBridge only supports External Card/Project ID.

![image](https://user-images.githubusercontent.com/60046135/95685556-3d6d3580-0c01-11eb-9ad8-83b64f3449fc.png)

However, if I want to quickly find out which project or card the record was created from, it requires too many actions to search.
I think it would be great to have another text field that would save the full link, as well as a button quick open on this link (opening a new tab in the browser).

![image](https://user-images.githubusercontent.com/60046135/95685796-cfc20900-0c02-11eb-84e0-b31db11b93c7.png)

What do you think about this improvement? Thanks!

**Comment by tobybatch** [comment]
Yes, it is!  I'm currently doing a full re-write, see here: https://github.com/tobybatch/ChromeKimaiBridge/issues/10#issuecomment-718963038

I'll incorporate it.
